consul_config.py.ctmpl

```diff
@@ -11,6 +8,10 @@ import json
     {{- end -}}
 {{- end -}}
 
+import os
```

Comment:
maybe worth a comment here to remind us not to add things before the `define`?

Reply:
Done